### PR TITLE
docs: new dr api document

### PR DESCRIPTION
**Pull request by changchaishi**
Create DR API doc for cooperation with Prof Yang's team.

### CODE REVIEW

File: apiary_dr.apib [hunk 1]
```diff
@@ -0,0 +1,300 @@
...
+            API-version: 0.1
+
+    + Attributes
+        + account: `NCKU` - Login Account
+        + password: `test` - Account's Password
+
++ Response 200 (application/json)
+    + Attributes
+        + id: `03f87a5a-a4e6-4536-ba6c-265c03ebd317` - User's UUID
+        + bearer: `OXBiogjCfdKb7m9CEOPhpbjjv0n5OtfltzgMJbSxbP7jVFY8XUNzHqkGidDumQSR` - Authentication Bearer Token
```
Comment (ttw225) [inline apiary_dr.apib:32]: 這邊實際上還會回 `is_aggregator`，雖然對他們來說應該沒差，但好像可以寫一下

```
+ `is_aggregator`: False (bool) - is aggregator or not
```

File: apiary_dr.apib [hunk 2]
```diff
@@ -0,0 +1,300 @@
...
+
++ Request BEMS OR AGGREGATOR GET BIDS
+    + Headers
+
+            API-version: 0.1
+            Authorization: `Bearer OXBiogjCfdKb7m9CEOPhpbjjv0n5OtfltzgMJbSxbP7jVFY8XUNzHqkGidDumQSR
+
+    + Parameters
+        + start_time (required, string)
+        + end_time (required, string)
```
Comment (ttw225) [inline apiary_dr.apib]: 這邊可以加個範例，順便說明一下我們的時間格式

```
### GET BIDS [GET /DR_bid{?start_time,end_time}]

+ Parameters
        + start_time: `2020-01-01 10:00` (required, string) - query start time (YYYY-MM-DD HH:mm)
        + end_time: `2020-01-01 11:00 ` (required, string) - query end time (YYYY-MM-DD HH:mm)
```

File: apiary_dr.apib [hunk 3]
```diff
@@ -0,0 +1,300 @@
...
+For BEMS: This route is to post dr request to Aggregator
+For Aggregator: This route is to accept and comfirm bems' submitted bids, so aggregator should provide
+uuids on those bids he/she wants to accept.
+</pre>
+
++ Request BEMS POST BID (application/json)
+    + Headers
+
+            API-version: 0.1
+            Authorization: `Bearer OXBiogjCfdKb7m9CEOPhpbjjv0n5OtfltzgMJbSxbP7jVFY8XUNzHqkGidDumQSR
```
Comment (ttw225) [inline apiary_dr.apib]: token 右邊少一個 ` XD

File: apiary_dr.apib [hunk 4]
```diff
@@ -0,0 +1,300 @@
...
+
+    + Attributes
+        + start_time: `2020-11-11 11:00:00` (string) - DR start time
+        + end_time: `2020-11-11 11:00:00` (string) - DR end time
```
Comment (ttw225) [inline apiary_dr.apib]: 看 end_time 要不要改成 12 點

File: apiary_dr.apib [hunk 5]
```diff
@@ -0,0 +1,300 @@
...
+            - `ca45ecc1-13a1-4a93-b047-857be806b764` (string)
+
++ Response 200 (application/json)
+    + Attributes (array, fixed-type)
+        + ()
+            + uuid: `36d87673-0daf-4ce2-917e-ec6d655d3c0a` (string)
+            + blockchain_url: `https://ropsten.etherscan.io/tx/0xc94ec346cea143d8aab07e3a23a9f24031a9a9ae5b9717d99cf32ac0f57a920d` (string) - url
+        + ()
+            + uuid: `ca45ecc1-13a1-4a93-b047-857be806b764` (string)
+            + blockchain_url: `https://ropsten.etherscan.io/tx/0x07178bc6d022a826d7f85b64f00b77bf878d04519cfbdc6e33aacd802b191435` (string) - url
```
Comment (ttw225) [inline apiary_dr.apib]: 這邊還不會回 blockchain url ~~

這樣應該回 200 or 400 就可以

**Review by ttw225 — CHANGES_REQUESTED** [review]

**Comment by changchaishi** [comment]
comments above are fixed in cda8fc6b097c1006235d7fc5b520006c87631c3f

In addition:
* Prettified document description rendered on `apiary.io`.
* fix extra typo by enabling spell checker on `.apib` file.

File: apiary_dr.apib (hunk 2 above)
Comment (ttw225) [reply apiary_dr.apib]: there is a space at the end of end_time, my fault XD
```
+ end_time: `2020-01-01 11:00 ` (required, string) - query end time (YYYY-MM-DD HH:mm)
-------------------------------^ extra white space
```

**Review by ttw225 — APPROVED** [review]
Please remove the extra white space and directly merge this branch~

File: apiary_dr.apib (hunk 2 above)
Comment (changchaishi) [reply apiary_dr.apib]: fixed in 32fe0bbad01bda67191a527171aa15703efa92b8

**Pull request merged**